jwst/associations/lib/rules_level3.py

```diff
@@ -352,21 +352,21 @@ def __init__(self, *args, **kwargs):
             ),
             Constraint(
                 [
-                DMSAttrConstraint(
-                name='bkgdtarg',
-                sources=['bkgdtarg'],
-                value=['T'],)
-                    ],
-                    reduce=Constraint.any
-                    ),
+                    DMSAttrConstraint(
+                        name='bkgdtarg',
+                        sources=['bkgdtarg'],
+                        value=['T'],)
+                ],
+                reduce=Constraint.any
+                                    ),
             Constraint(
                 [
-                DMSAttrConstraint(
-                name='mir_bkgdtarg',
-                sources=['exp_type'],
-                value=['mir_mrs'],)
-                    ],
-                    reduce=Constraint.any
+                    DMSAttrConstraint(
+                        name='allowed_bkgdtarg',
+                        sources=['exp_type'],
+                        value=['mir_mrs','nrs_ifu'],)
```

Comment:
I wonder if this is too restrictive. There are other EXP_TYPE's that are allowed to have associated background targets, such as mir_lrs-fixedslit, nrs_fixedslit, and eventually the imaging modes too (e.g. nrc_image, mir_image). Does there need to be a constraint involving EXP_TYPE at all or can we just go with the `bkgdtarg` constraint above?

Reply:
Good idea and I like simplification. So I tried it. Unfortunately it breaks several spec3 associations e.g. jw00623-o002_spec3_001_asn.json.

Reply:
So if removing this constraint doesn't work, then we should at least expand the list of exp_type values to also include "mir_lrs-fixedslit" and "nrs_fixedslit".

Reply:
Updated the allowed_exptypes to include  "mir_lrs-fixedslit" and "nrs_fixedslit". nrs_fixedslit is already in the test suite but I'll need to find an example of mir_lrs-fixedslit for testing.